src/PostgREST/Error.hs

```diff
@@ -166,6 +179,7 @@ httpStatus authed (P.SessionError (H.ResultError (H.ServerError c _ _ _))) =
     "42883"   -> HT.status404 -- undefined function
     "42P01"   -> HT.status404 -- undefined table
     "42501"   -> if authed then HT.status403 else HT.status401 -- insufficient privilege
+    'P':'T':n -> fromMaybe HT.status400 (HT.mkStatus <$> readMaybe n <*> pure m)
```

Comment:
If `n` can't be read as a number, maybe we should return a 500 error rather than 400 since it's a problem internal to the server rather than a problem with the request.